aws-pinpoint-core/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+plugins {
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.6.10"
```

Comment:
latest stable version is [1.8.10](https://plugins.gradle.org/plugin/org.jetbrains.kotlin.plugin.serialization/1.8.10). Shall we investigate and update ?

Reply:
Yes, we can do that a follow up.